packages/twenty-front/src/modules/object-record/record-filter/utils/turnObjectDropdownFilterIntoQueryFilter.ts

```diff
@@ -24,6 +24,128 @@ export type ObjectDropdownFilter = Omit<Filter, 'definition'> & {
   };
 };
 
+const applyEmptyFilters = (
+  operand: ViewFilterOperand,
+  correspondingField: Pick<Field, 'id' | 'name'>,
+  objectRecordFilters: RecordGqlOperationFilter[],
+  filterType:
+    | 'URL'
+    | 'Address'
+    | 'String'
+    | 'Number'
+    | 'Date'
+    | 'Relation'
+    | 'CompositeString' = 'String',
```

Comment:
think you should use `Pick<FilterType, 'LINK' | 'ADDRESS' | 'FULL_NAME' | ...>`

Reply:
We should work only with field types here, not introduce a new type